lib/transform.js

```diff
@@ -11,13 +11,27 @@ const CustomSerializer = require('./serializer');
  */
 
 module.exports = class Transform {
-    constructor(handleTags) {
+    constructor(handleTags, maxTemplates) {
         this.handleTags = handleTags;
-        this.parse = memoize(parse5.parse.bind(null), { length: 1, max: 20 });
-        this.parseFragment = memoize(parse5.parseFragment.bind(null), {
+
+        const commonMemoizeOptions = {
             length: 1,
-            max: 20
-        });
+            maxTemplates
+        };
+
+        this._ignore = memoize(
```

Comment:
Pls explain how memoization will help here if baseTemplate will be always different?

Reply:
Same goest to the `_restore`

Reply:
It is not relevant regarding new changes
That is not needed anymore